README.md

```diff
@@ -22,4 +21,4 @@ that make it easy by automatically tracking dependencies between computations.
 If you are familiar with Salsa-rs, you'll see many things that are familiar and a few
 slightly more generic, and moved away from the database-oriented naming:
 - **derived queries** => **`@derived` functions**
-- **query groups** => **`@component`s**
+- **query group** => **`Runtime`**
```

Comment:
While true, we might at some point want to extend the README to explain that there is supposed to be only a single runtime, rather than several different query groups?

Reply:
Yesh, definitely agreed that we need a much improved readme :C